app/models/jobstatusdata.rb

```diff
@@ -27,6 +27,7 @@ def initialize(info, cluster=OODClusters.first.id.to_s, extended=false)
     self.account = info.accounting_id
     self.status = status_label(info.status.state.to_s)
     self.cluster = cluster
+    self.cluster_title = build_title(cluster)
```

Comment:
I think it would be easiest to just replace this with:

```ruby
self.cluster_title = OODClusters[cluster].metadata.title ||  cluster.titleize
```

as you already assume `OODClusters[cluster]` exists below.